spec/classes/repo_apt_spec.rb

```diff
@@ -53,7 +53,7 @@
           end
         end
 
-        context 'Specific Gluster release 3.9' do
+                context 'Specific Gluster release 3.9' do
```

Comment:
Nit: could you clean up the intendenting here?

Reply:
Sure thing, sorry!